src/pages/blog/template/index.js

```diff
@@ -0,0 +1,417 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import { localize, WithIntl } from 'components/localization'
+import Layout from 'components/layout/layout'
+import { SectionContainer, SEO, Show, Box, Flex } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import device from 'themes/device'
+
+const HeroContainer = styled.div`
+    display: flex;
+    width: 100%;
+    max-width: 144.4rem;
+    padding: 7.8rem 12.4rem 14.6rem 12rem;
+    height: 56.2rem;
+    background-color: var(--color-grey-8);
+    margin: auto;
+    margin-bottom: 8.6rem;
+
+    @media (max-width: 1350px) {
+        height: 50rem;
+    }
+    @media ${device.tabletL} {
+        flex-direction: column;
+        margin-bottom: 0;
+        padding: 38px 16px 0;
+        height: auto;
+        background-image: linear-gradient(#f2f3f4 80%, #ffffff 20%);
+    }
+`
+const HeroImageContainer = styled(Box)`
+    @media ${device.tabletL} {
+        margin: auto;
+    }
+
+    > img {
+        max-width: 100%;
+        height: auto;
+    }
+`
+const WriterContainer = styled(Box)`
+    display: flex;
+    align-items: center;
+`
+const PublishDate = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
+const MinsToRead = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
+const WriterImage = styled.div`
+    width: 4.8rem;
+    height: 4.8rem;
+    border-radius: 5rem;
+    margin-right: 8px;
+
+    @media ${device.tabletL} {
+        width: 4rem;
+        height: 4rem;
+
+        > img {
+            max-width: 100%;
+            height: auto;
+        }
+    }
+`
+const AuthorText = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
+const WrittenbyText = styled(Text)`
+    color: var(--color-grey-5);
+    font-size: var(--text-size-xxs);
```

Comment:
These size variables should be deprecated, you can use `px` here or pass in as props. Can pass in `color` as props too.